docs/en/advanced/Client-Logging.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: global
+title: Client Logging
+nickname: Client Logging
+group: Advanced
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page summarizes how to find client-specific logs and modify logging levels in order to debug
+client-sided issues with your Alluxio installation.
+
+## Log Location
+
+Depending on which compute framework you are using, the location of the logs can vary. We
+recommend looking at the specific documentation for your compute framework to determine where to
+find alluxio cient logs.
+
+Here are links to compute frameworks with information where logs may be found and how to configure
+
+- [Apache Spark]({{ site.baseurl }}{% link en/compute/Compute-Spark.md %})
+- [Apache Hadoop]({{ site.baseurl }}{% link en/compute/Compute-Hadoop-MapReduce.md %})
+- [Apache HBase]({{ site.baseurl }}{% link en/compute/Compute-HBase.md %})
+- [Apache Hive]({{ site.baseurl }}{% link en/compute/Compute-Hive.md %})
+- [Presto]({{ site.baseurl }}{% link en/compute/Compute-Presto.md %})
+
+However, by setting the `alluxio.logs.dir` property you can modify where the logs are stored.
```

Comment:
This only works if they are using our log4j.properties file, i.e. when using alluxio shell commands. Compute applications would usually have their own log4j.properties, and Alluxio logs will go to the same file as the compute application's own logs. This is nice because then users only have to check one log, and Alluxio logs are surrounded by the context of what is happening within the compute application.

The alluxio-specific side of the compute logging is just that Alluxio uses log4j, so users can find its logs in the same place that they find their other application logs, and users can modify their existing log4j.properties to change Alluxio's log level.

This is an example of how `Spark` talks about getting its logs out of `YARN`: http://spark.apache.org/docs/latest/running-on-yarn.html#debugging-your-application

Reply:
I think what I might do is for each compute framework, add a small section on logging with a link that points to a framework-specific piece of documentation on logging, and for these links, make them anchored to the logging section within our docs.

What do you think?

Reply:
Sounds good